domains/transfer/Transfer.API/API/Transfer/Api/Exceptions/EntityNotFoundException.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace API.Transfer.Api.Exceptions;
+
+public class EntityNotFoundException : NotFoundException
```

Comment:
Can we move these execptions out in lib. Because we have them in several subsystems.